### PR TITLE
Update Trace to Logs docs in configure-tempo-data-source.md

### PR DESCRIPTION
**Pull request by amanos1022**
Trace to logs documentation is unclear on a couple things. I think the context I've added will help others get things working a little easier.

### CODE REVIEW

**Comment by Eve832** [comment]
@knylander-grafana can you please review this PR?

**Comment by GrafanaWriter** [comment]
@knylander-grafana - I believe you were pinged while on PTO; can you please take a look?

**Review by knylander-grafana — APPROVED** [review]

File: docs/sources/datasources/tempo/configure-tempo-data-source.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ There are two ways to configure the trace to logs feature:
...
 
 1. Set start and end time shift. As the logs timestamps may not exactly match the timestamps of the spans in trace it may be necessary to search in larger or shifted time range to find the desired logs.
-1. Select which tags to use in the logs query. The tags you configure must be present in the spans attributes or resources for a trace to logs span link to appear. You can optionally configure a new name for the tag. This is useful for example if the tag has dots in the name and the target data source does not allow using dots in labels. In that case you can for example remap `http.status` to `http_status`.
+1. Select which tags to use in the logs query. The tags you configure must be present in the span's attributes or resources for a trace to logs span link to appear. You can optionally configure a new name for the tag. This is useful for example if the tag has dots in the name and the target data source does not allow using dots in labels. In that case you can for example remap `http.status` (the span attribute) to `http_status` (the data source field). "Data source" in this context can refer to Loki, or another log data source.
```
Comment (knylander-grafana) [inline docs/sources/datasources/tempo/configure-tempo-data-source.md:58]: The content looks good. Just found some missing commas. See the suggestion below. 
```suggestion
1. Select which tags to use in the logs query. The tags you configure must be present in the span's attributes or resources for a trace to logs span link to appear. You can optionally configure a new name for the tag. This is useful, for example, if the tag has dots in the name and the target data source does not allow using dots in labels. In that case, you can for example remap `http.status` (the span attribute) to `http_status` (the data source field). "Data source" in this context can refer to Loki, or another log data source.
```

**Review by knylander-grafana — APPROVED** [review]
Approving with one minor suggestion. Thank you for updating the doc. Sorry it took me a little bit to get to it.

**Comment by amanos1022** [comment]
> Approving with one minor suggestion. Thank you for updating the doc. Sorry it took me a little bit to get to it.

No problem, thanks for taking a look. Committed your suggestion.

**Review by knylander-grafana — APPROVED** [review]
Thank you for adding this information. It helps clarify the content. I don't have permission to merge the changes.

**Pull request merged**